src/client/containers/CartPage/CartPage.Container.js

```diff
@@ -1,48 +1,77 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 import React from 'react';
 import './CartPage.Style.css';
-import { useParams } from 'react-router-dom';
 import Loader from '../../components/Loader/Loader.component';
 import ShoppingItem from '../../components/ShoppingItem/ShoppingItem';
-import DeliveryInfo from '../../components/DeliveryInfo/DeliveryInfo.component';
 import TotalPrice from '../../components/TotalPriceCard/TotalPriceCard.component';
 import ContactForm from '../../components/ContactForm/ContactForm.component';
 import ButtonComponent from '../../components/Button/Button.component';
-import { useFetchApi } from '../../hooks/UseFetchApi';
 import { useFirebase } from '../../firebase/FirebaseContext';
 import { PropTypes } from 'prop-types';
+import { useShoppingCartContext } from '../../context/shoppingCart/shoppingCartContext';
+import { useHistory } from 'react-router-dom';
+import DeliveryInfoV2 from '../../components/DeliveryInfo/DeliveryInfoV2.component';
+import { useFetchApi } from '../../hooks/UseFetchApi';
 
 const CartPageContainer = ({ isAuthenticated }) => {
   const [cartItem, setCartItem] = React.useState([]);
-  const [user, setUser] = React.useState({});
-  const [userId, setUserId] = React.useState('');
+  const [IsLoading, setIsLoading] = React.useState(true);
```

Comment:
variables should be `camelCase`. (have to update that where it is used)
```suggestion
  const [isLoading, setIsLoading] = React.useState(true);
```

Reply:
👍